src/fontra/views/editor/editor.js

```diff
@@ -2272,12 +2288,15 @@ export class EditorController {
       }
       const newAnchorIndex = instance.anchors.length - 1;
       this.sceneController.selection = new Set([`anchor/${newAnchorIndex}`]);
-      return "Add Anchor";
+      return translate("action.add-anchor");
     });
   }
 
   async doAddEditAnchorDialog(anchor = undefined, point = undefined) {
-    const titlePrefix = anchor ? "Edit" : "Add";
+    const titleDialog = anchor
```

Comment:
Similarly, suggest to put the switch inside the `translate()` call, so we don't have to repeat it.